core/src/test/java/org/infinispan/api/NonTxCacheTest.java

```diff
@@ -58,7 +56,7 @@ public void testRemove() throws Exception {
 
    @Override
    protected void createCacheManagers() throws Throwable {
-      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(cacheMode, false);
+      ConfigurationBuilder builder = getDefaultClusteredCacheConfig(CacheMode.REPL_SYNC, false);
```

Comment:
yes, it is testing an issue with the old algorithm. Triangle doesn't suffer the same issue and only the replicated cache does.